app.js

```diff
@@ -82,9 +88,9 @@ function createApp(config) {
   app.use(requestId())
   app.use(cachingMiddleware(caching()))
 
-  app.use(logger('dev'))
+  app.use(morgan('dev'))
```

Comment:
Consider using the standard Apache log format - includes useful information in the logs such as remote IP address and the user-agent HTTP request header.
app.use(morgan('combined'));